### PR TITLE
Setting up structure for pypi packaging

### PR DESCRIPTION
**Pull request by Ckoelewyn**
Hey team, this is the structure needed in the repo for #17

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    # TODO: Decide name for package
+    name="TO BE DECIDED",
```
Comment (Ckoelewyn) [inline setup.py:8]: We should discuss this at standup today

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (Ckoelewyn) [inline LICENSE:1]: Algo 2 went MIT and I didn't see a reason why we shouldn't as well. We can discuss this at standup today if anyone has concerns.

**Review by kianapaz — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Copyright (c) 2022 [fullname]
```
Comment (sfaigan) [inline LICENSE:3]: Can we put all our names?

File: setup.py (hunk 1 above)
Comment (sfaigan) [inline setup.py:8]: "UVic Software Engineering Course Scheduler" or "Course Scheduler" if it must match the package name

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    version="0.0.1",
+    author="Corey Koelewyyn",
+    author_email="Corey.Koelewyn@gmail.com",
+    description="A scheduling package for UVIC SENG499 capstone",
```
Comment (sfaigan) [inline setup.py:12]: nit: "A course scheduler for the software engineering program at UVic. Built for the SENG 499 Summer 2022 project."

**Review by sfaigan — COMMENTED** [review]
Looks good! Thanks for setting this up!

General feedback:
- Rename _algo1_ folder to _coursescheduler_
- Rename _algo1.py_ to _scheduler.py_, or _coursescheduler.py_ if the convention is that it matches the package name
- Remove _/tests/\_\_pycache\_\__

**Pull request merged**